OutOfSchool/OutOfSchool.WebApi/Services/RatingService.cs

```diff
@@ -348,17 +354,17 @@ private async Task<bool> EntityExists(long id, RatingType type)
             }
         }
 
-        private IEnumerable<RatingDto> GetUsersAsync(IEnumerable<RatingDto> ratingDtos)
+        private async Task<IEnumerable<RatingDto>> AddUserInformation(IEnumerable<RatingDto> ratingDtos)
         {
             var ratingDtosList = ratingDtos.ToList();
 
-            var newUsers = parentRepository.GetUsersByParents(ratingDtosList.Select(r => r.ParentId).Distinct());
+            var parents = await parentRepository.GetParentsByIds(ratingDtos.Select(rating => rating.ParentId).Distinct()).ConfigureAwait(false);
 
             for (int i = 0; i < ratingDtosList.Count; i++)
             {
```

Comment:
You can use ratingDtoList.ForEach(...) to update elements inside the collection

Reply:
I changed it to Select for do not cast IEnumerable to List here.